apps/web-giddh/src/app/daybook/daybook.component.html

```diff
@@ -68,6 +69,13 @@
                                 (click)="exportDaybook()"
                                 [disabled]="!((daybookData$ | async)?.entries?.length > 0)">{{commonLocaleData?.app_export}}</button>
                         </div>
+                        <div class="form-group cursor-pointer desktop-view float-right">
+                                <div class="advance-icon align-items-center btn btn-primary"
+                                    [tooltip]="commonLocaleData?.app_advance_search" (click)="onOpenAdvanceSearch()">
+                                    <span class="pd-r05">Filter</span>
```

Comment:
Do add multi-lingual labels and avoid hard coded values

Reply:
done